lib/account.rb

```diff
@@ -0,0 +1,66 @@
+require 'csv'
+
+module Bank
+	class Account
+
+		attr_accessor :balance, :account_id, :date_opened
+
+		def initialize(account_id, balance, date_opened)
+			@account_id = account_id
+			@balance = balance.to_i
+			check_balance
+			@min_balance = nil
+			@date_opened = DateTime.strptime(date_opened, "%Y-%m-%d %H:%M:%S %z")
+
+			puts "The balance is #{@balance}."
+		end
+
+		def check_balance
+			if @balance.to_i < @min_balance.to_i
```

Comment:
Since you're using `@min_balance` here but not setting it until after this method is called, what would this be checked against?